interp/run.go

```diff
@@ -1148,7 +1136,7 @@ func call(n *node) {
 			j := n.findex + i
 			ret := n.child[0].typ.ret[i]
 			callret := n.anc.val.(*node).typ.ret[i]
-			if callret.cat == interfaceT && len(callret.field) > 0 && ret.cat != interfaceT {
+			if isInterfaceSrc(callret) && !isEmptyInterface(callret) {
```

Comment:
Are you sure about this one? it looks like the part "ret.cat != interfaceT" disappeared.

Reply:
I added the missing test.